qa/rpc-tests/wallet_z_listreceivedbyaddress.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python2
+# Copyright (c) 2018 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import connect_nodes_bi, initialize_chain_clean, start_nodes, stop_nodes, \
+    wait_and_assert_operationid_status, wait_bitcoinds
+
+from decimal import Decimal
+
+class WalletZListReceivedByAddressTest (BitcoinTestFramework):
```

Comment:
This test doesn't seem to be related to the subject of the PR.

Reply:
force-pushed fix